dlib/travis/build-and-test.sh

```diff
@@ -7,11 +7,16 @@ if [ -n "${MATRIX_EVAL+set}" ]; then
     eval "${MATRIX_EVAL}"
 fi
 
+CXX_FLAGS="-std=c++11"
+if [ ! -z ${CXXFLAGS+x} ]; then
```

Comment:
```suggestion
if [ ! -z ${CXXFLAGS+set} ]; then
```
Do this though, so it's the same style as the one above.